addons/account/models/account_payment.py

```diff
@@ -16,6 +16,7 @@ class AccountPayment(models.Model):
     move_id = fields.Many2one(
         comodel_name='account.move',
         string='Journal Entry', required=True, readonly=True, ondelete='cascade',
+        index='btree_not_null',
```

Comment:
A `required=True` field cannot contains NULLs, it should be a simple `index=True`

Reply:
Fixed